### ISSUE TITLE
Strange behaviour in telemetry page

### ISSUE THREAD

**Issue by MicheleVilla** [opentx/opentx#2714]
Taranis (not plus)
opentx 2.1.2N418
Italian language
X8R receiver binded

Telemetry page
Screen1 
option: Value

When scroll parameter the showed value are
.....
.....
.....
Gv9
  OFF
ABSM
NO
(Blank)
(Blank)
RSSI
RSSI-
RSSI+
RxBt
RxBt-
RxBt+
SWR
SWR-
SWR+
"-" 
"+"

If choise "NO"  in telemetry page is showed T1
If choise first Blank space in telemetry page is showed T2
If choise second Blank space in telemetry page is showed T3

Tx battery is no available

Possible is a translation bug?

Tanks

**Comment by kilrah** [comment]
Yes there must be errors in the translation.

**Comment by MicheleVilla** [comment]
I have found some difference from it.h.txt to en.h.txt
I will try to edit file later

**Comment by projectkk2glider** [comment]
The Italian (and possibly others) have a lot of changes missing in `TR_VSRCRAW`  which is now assembled from multiple other strings.

**Comment by projectkk2glider** [comment]
@MicheleVilla let me fix that first and they you can make your translations.

**Comment by projectkk2glider** [comment]
@MicheleVilla It should be fixed now.

**Comment by MicheleVilla** [comment]
Thank you for your fixes
Now I will start to fix other translations and test all

**Comment by MicheleVilla** [comment]
Now seems all Ok

**Comment by MicheleVilla** [comment]
Update:
Taranis  with X8R
firmware compiled from source  (today 17/08)
both simulator and firmware installed on radio

On Simulator is Ok but on radio the last option are
SWR
SWR-
SWR+
"-"
"+"

What means two last option?
are a bug?

**Comment by projectkk2glider** [comment]
Yes, can you upload your eepe? Have you tried English language, does it have the same problem?

**Comment by MicheleVilla** [comment]
Try with other memory an new model
all ok 
problem is only with a singol model
I check but I think there is no other bug
Do you want the eepe?

The telemetry parameter are memorized the first time radio is connected to receiver and remains in memory
Is possible to reset the data of telemetry ?
Or the procedure is different?

before post is better to make different test......

**Comment by projectkk2glider** [comment]
>Is possible to reset the data of telemetry ?

If you mean to clear all detected telemetry values, then there is no such function avalilable (yet). @bsongis maybe we should add a menu option to delete all sensors?

**Comment by bsongis** [comment]
Yes why not!
Le 18 août 2015 5:44 PM, "Damjan Adamic" <notifications@github.com> a
écrit :

**Comment by WZ9V** [comment]
Yes please.  It would come in handy when dealing with systems that goof up and fill up all the slots.  I just had it happen today with TauLabs and now I need to dump all the discovered sensors to test the fix.  Probably won't get used much once things are sortd out but for now it's almost a necessity until everything else gets updated.

**Comment by bsongis** [comment]
Then your Taulabs firmware doesn't fulfill S.PORT protocol rules. In 2.1.1
you can ignore the telemetry ID, it will hide your sensor bug.

**Comment by kilrah** [comment]
Just do the tests on a throwaway new model that you can just delete and make a new one until everything's sorted.

**Comment by WZ9V** [comment]
The Taulabs folks have corrected the problem and it will be in the next version.  I tested their fix last night.  I believe the approach they used was to have the FC use one device ID.  Now I just get the expected values without flooding all the slots.  That branch is bleeding edge so I'll just disable telemetry until it's pushed out as a stable release.  Kilrah, that's the approach I took, just deleted the model made a new one and then copied it for the test so I could delete it if telemetry got flooded again.
